7-gce/tox.ini

```diff
@@ -8,17 +8,23 @@ deps =
   -rrequirements.txt
   -rrequirements-dev.txt
 commands =
-  nosetests \
-    --with-flaky \
-    --no-success-flaky-report \
-    --with-coverage \
-    --cover-package bookshelf \
-    {posargs:-a '!e2e'}
+  py.test --cov=bookshelf --no-success-flaky-report -m "not e2e" {posargs: tests}
```

Comment:
I'm not going to rip tox out of the individual steps. I still think that most python developers will expect to see tox. We'll use nox to orchestrate running all of them together though.